sw/device/silicon_creator/lib/drivers/ibex.c

```diff
@@ -27,68 +27,121 @@ uint32_t ibex_fpga_version(void) {
   return abs_mmio_read32(kBase + RV_CORE_IBEX_FPGA_INFO_REG_OFFSET);
 }
 
-void ibex_addr_remap_0_set(uint32_t matching_addr, uint32_t remap_addr,
-                           size_t size) {
+size_t ibex_addr_remap_slots(void) { return RV_CORE_IBEX_PARAM_NUM_REGIONS; }
```

Comment:
Oh nice, this is  already rendered via the original register file, no need for DT :)